opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/resources/mockedsandbox/restrecord-nonhappy/embedded/multi-sca/accounts/sandbox/mappings/mapping-v1-accounts-without-balance.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Why do we need this file in **non-happy** path tests?

Reply:
because now some tests now pass withBalance=false wich was not found before and thus failed.

Reply:
Existing fixture must be modified